wiki/Rules/Visual_Content_Considerations/en.md

```diff
@@ -55,4 +55,4 @@ If you are unsure about an image that is going to be included in a beatmap in an
 
 **The result of any vote is final**, unless the image is significantly altered in an attempt to resolve any potentially problematic elements raised during discussion.
 
-At all times, the [osu!team](/wiki/People/The_Team) reserves the right to unilaterally declare any piece of content either valid or invalid at their discretion, regardless of any previous voting held on the matter.
+At all times, the voting teams reserve the right to unilaterally declare any piece of content either valid or invalid at their discretion, regardless of any previous voting held on the matter.
```

Comment:
also reads weird. not sure what a "voting team" is

Reply:
to be frank, I'm not sure what `the osu!team` is either, considering that not all people from `People/The team`, and not all user groups listed there, have access to voting. in this case, `the osu!team`, or `the osu! team`, becomes `the people and teams who vote, but not necessarily these who are on the page`, which is about as ambiguous as [other usages of it](https://github.com/search?q=osu%21team+repo%3Appy%2Fosu-wiki+filename%3Aen.md&type=Code&l=&l=)

Reply:
changed it to "the osu! team" but the issue still stands